tasks/recover/add_domain.yml

```diff
@@ -1,38 +1,43 @@
 - ovirt_hosts_facts:
       pattern: "status=up and datacenter={{ storage['dr_' + dr_target_host + '_dc_name'] }}"
       auth: "{{ ovirt_auth }}"
+- block:
+  - name: "Check for available hosts"
+    fail: msg="No hosts available"
+    when: ovirt_hosts.0 is undefined
+- block:
```

Comment:
I think this change below isn't needed, it will fail above, so the code below won't be executed.

Reply:
```
TASK [ovirt-ansible-disaster-recovery : Check for available hosts] ********************************************************************************
fatal: [localhost]: FAILED! => {"changed": false, "msg": "No hosts available"}
...ignoring

TASK [ovirt-ansible-disaster-recovery : include_tasks] ********************************************************************************************
included: /home/bzlotnik/Development/ovirt/ovirt-ansible-disaster-recovery/tasks/recover/add_nfs_domain.yml for localhost

TASK [ovirt-ansible-disaster-recovery : Add NFS storage domain] ***********************************************************************************
fatal: [localhost]: FAILED! => {"msg": "The task includes an option with an undefined variable. The error was: list object has no element 0\n\nThe error appears to have been in '/home/bzlotnik/Development/ovirt/ovirt-ansible-disaster-recovery/tasks/recover/add_nfs_domain.yml': line 1, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n- name: Add NFS storage domain\n  ^ here\n"}
...ignoring

It looks like it does try to execute, leave the conditional block as it is?
```

Reply:
Hmm, right, that's because `dr_ignore_error_recover` is `True`.